src/propsheet/PropSheetHandler.cpp

```diff
@@ -217,7 +217,7 @@ class ConsolePropertySheetHandler WrlFinal : public RuntimeClass<RuntimeClassFla
                         {
                             // Second expensive portion of this method -- cracks the PE header of the .lnk file target
                             // if it's an executable
-                            SHFILEINFO sfi = { 0 };
+                            SHFILEINFO sfi = {};

```

Comment:
Same comment here, can we leave this zero-initialized?
```suggestion
                            SHFILEINFO sfi = { 0 };

```

Reply:
This is 0 initialized, in c++, except this is a struct/array of pointers. {} is zero-initialization in c++ without having to worry about pointers.

Reply:
@DHowett-MSFT do you agree?